test/NuGet.Core.Tests/NuGet.DependencyResolver.Core.Tests/FindPackageTests.cs

```diff
@@ -165,7 +165,7 @@ public async Task FindLibraryEntryAsync_LogsOnlyPackages(LibraryDependencyTarget
         }
 
         [Fact]
-        public async Task FindPackage_VerifyMissingListedPackageThrowsNotFound()
+        public async Task FindPackage_VerifyMissingListedPackageThrowsNotFoundAndLogsErrorAsync()
```

Comment:
Nit: The underscore helps separate what we're testing and what the outcome is in the test title.

```suggestion
        public async Task FindPackage_VerifyMissingListedPackage_ThrowsNotFoundAndLogsErrorAsync()
```